dohq_artifactory/admin.py

```diff
@@ -93,9 +94,14 @@ def _create_and_update(self):
             headers={"Content-Type": "application/json"},
             auth=self._auth,
         )
-        r.raise_for_status()
-        rest_delay()
-        self.read()
+        try:
+            r.raise_for_status()
```

Comment:
You can see repeated parts of code. May be we can use one function to do this?
```python
def raise_errors(r):
    try:
        r.raise_for_status()
   except HTTPError as e:.
       if ....
       else
           raise e

# Use instead of
# r.raise_for_status()
raise_errors(r)
rest_delay()
self.read()
```